grype/matcher/bitnami/matcher.go

```diff
@@ -0,0 +1,27 @@
+package bitnami
+
+import (
+	"github.com/anchore/grype/grype/match"
+	"github.com/anchore/grype/grype/matcher/internal"
+	"github.com/anchore/grype/grype/pkg"
+	"github.com/anchore/grype/grype/vulnerability"
+
+	syftPkg "github.com/anchore/syft/syft/pkg"
+)
+
+type Matcher struct{}
+
+func (m *Matcher) PackageTypes() []syftPkg.Type {
+	return []syftPkg.Type{syftPkg.BitnamiPkg}
+}
+
+func (m *Matcher) Type() match.MatcherType {
+	return match.BitnamiMatcher
+}
+
+func (m *Matcher) Match(store vulnerability.Provider, p pkg.Package) ([]match.Match, []match.IgnoredMatch, error) {
+	// Bitnami packages' metadata are built from the package URL (instead of CPE, hence CPE match is not supported)
+	// ref: https://github.com/anchore/syft/blob/main/syft/pkg/bitnami.go#L3-L13
+	// ref: https://github.com/anchore/syft/blob/main/syft/pkg/cataloger/bitnami/package.go#L18-L45
+	return internal.MatchPackageByEcosystemAndCPEs(store, p, m.Type(), false)
```

Comment:
@willmurphyscode should we include some mechanisms to match based on the pURL?

Reply:
Thanks for the context @willmurphyscode!
FYI: I was assigned a top priority task so you should expect a couple of days delay on my work here. I'll probably resume it next Monday.

Reply:
I did a 1st approach at https://github.com/anchore/grype/pull/2538/commits/b6e185830f3016800514356def6486ad20b7f3d9